data/simple-icons.json

```
@@ -3835,8 +3835,9 @@
 	},
 	{
 		"title": "DaisyUI",
```

```suggestion
		"title": "daisyUI",
```

See https://daisyui.com/brand/#daisyui-name

---

Yes, I forgot to change